app/questionnaire/relationship_router.py

```diff
@@ -29,7 +30,7 @@ def __init__(
         self.unrelated_no_answer_values = unrelated_no_answer_values or []
         self.path = self._relationships_routing_path()
 
-    def can_access_location(self, location: RelationshipLocation) -> bool:
+    def can_access_location(self, location: Location | RelationshipLocation) -> bool:
```

Comment:
when would this be a `Location`?

Reply:
ahh.. I see the issue, the relationship router is part of relationship question inheriting from question from blockhandler where the location is either. Hmmm, not sure if better to stay as is or type ignore

Reply:
Will keep to RelationshipLocation and type ignore the error to keep consistent to what we did yesterday